bin/upload-staticfiles.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -exo pipefail
+
+source docker/bin/set_git_env_vars.sh
+
+if [[ "$BRANCH_NAME" != "prod" ]]; then
+    # nothing to do if not prod deployment
+    exit 0
+fi
+
+CONTAINER_NAME="bedrock-${BRANCH_AND_COMMIT}"
+
+rm -rf static
+# have to rerun staticfiles without symlinks or we just copy broken symlinks
+docker-compose run --name "$CONTAINER_NAME" release docker/bin/build_staticfiles.sh --nolink
+docker cp "${CONTAINER_NAME}:/app/static" ./static
+docker rm -f "$CONTAINER_NAME"
+
+cd static
+for BUCKET in stage prod; do
+    # hashed filenames
+    aws s3 sync \
+        --exclude "*" \
+        --include "*.????????????.*" \
+        --acl public-read \
+        --cache-control "max-age=315360000, public, immutable" \
+        --profile bedrock-media \
```

Comment:
any reason for using `--profile` vs the `AWS_PROFILE` env var?

Reply:
that I didn't know about `AWS_PROFILE` and was just copying what IRL did:

https://github.com/mozilla/irlpodcast/blob/91b8bbb35da7f9ff3b1e844b7ce103bfd43eca0a/Jenkinsfile#L17

Reply:
Happy to switch it if one is better than the other though.